backend/src/models/user.go

```diff
@@ -56,3 +56,14 @@ type User struct {
 	RSVP              []Event   `gorm:"many2many:user_event_rsvps;constraint:OnUpdate:CASCADE,OnDelete:CASCADE;" json:"-" validate:"-"`
 	Waitlist          []Event   `gorm:"many2many:user_event_waitlists;constraint:OnUpdate:CASCADE,OnDelete:CASCADE;" json:"-" validate:"-"`
 }
+
+// TODO: Should we change error message for missing required fields?
```

Comment:
@DOOduneye ?

Reply:
The error messages given by the validators seemed kinda wordy and unreadable. For example if a required field like College is missing, the error message is `"Key: 'CreateUserRequestBody.College' Error:Field validation for 'College' failed on the 'required' tag"` and we feel like this can be better written. Thoughts?